docs/HowToRunSTDKSample.md

```diff
@@ -0,0 +1,254 @@
+
+## Contents
+* [Prerequisites](#prerequisites)
+* [Get TizenRT Source](#get-tizenrt-source)
+* [Create device identity](#create-device-identity)
+* [Register Device Profile on SmartThings Devworkspace](#register-device-profile-on-smartthings-devworkspace)
+* [Build and Flash](#build-and-flash)
+* [Try to do EasySetup and Control](#try-to-do-easysetup-and-control)
+* [How to develop your STDK device](#how-to-develop-your-stdk-device)
+
+## Prerequisites   
+This section will describe the pre-requisite steps to install and build STDK for the ESP-WROVER-KIT board.   
+
+* Board specific prerequisites [[Details]](../README.md#supported-board--emulator).   
+* Toolchain installation [[Details]](https://smartthings.developer.samsung.com/docs/devices/direct-connected-devices/setup-environment.html).   
+
+### Get TizenRT Source   
+#### Clone TizenRT source code.   
+```bash
+git clone https://github.com/Samsung/TizenRT.git
+cd TizenRT
+TIZENRT_BASEDIR="$PWD"
+```
+#### Apply patch for STDK.   
++ Update submodule   
+```bash
+git submodule update --init "external/stdk/st-device-sdk-c"
+git submodule update --init "external/libsodium/libsodium"
+```
++ Apply patch for stdk   
+```bash
+cd external/stdk/st-device-sdk-c
+git apply ../patch/0001-port-net-fix-ssl-issue-on-TizenRT.patch
+```
+
+## Create device identity   
+Your IoT device needs authentication in order to connect to the SmartThings Cloud. These data usually reside in file format. To create these  authentication data, you must know your MNID in the personal information window of [Developer Workspace](https://smartthings.developer.samsung.com/). Please sign in with Samsung Account and then check your MNID first.   
+
+Open a terminal window and run the following stdk-keygen command to create a Device Identity.   
+
+```bash
+$ cd external/stdk/st-device-sdk-c/tools/keygen/linux
+$ ./stdk-keygen -h
+usage: stdk-keygen -m MNID [-f firmware_version]
+
+$ ./stdk-keygen -m **** -f V201910       # replace '****' with your MNID
+Go Device Identity of Developer Workspace.
+
+Serial Number:
+┌──────────┬──────────┐
+│ STDK**** │ ce**2**3 │
+└──────────┴──────────┘
+
+Public Key:
+1D********a21F********8WwP********yU/n8vFvM=
+
+$ tree
+.
+├── ed25519.pubkey
+├── ed25519.seckey
+└── output_STDK****ce**2**3
+    └── device_info.json
+```
+Copy the Serial Number and Public Key after running the command on your computer. You will need to upload those values to the SmartThings Cloud via [Developer Workspace](https://smartthings.developer.samsung.com/).
+
+After you create a device identity with a command with an option like above, You can get the ready-to-use device_info.json file directly.   
+
+## Register Device Profile on SmartThings Devworkspace   
+### Sing in to [Devworkspace](https://smartthings.developer.samsung.com/).   
+Select 'Sing In' where right-top on the page.   
+![Select Sign In](./media/STDK_Selection_Sign_in.png)   
+![Select Sign In](./media/STDK_Singin_devworkspace.png)   
```

Comment:
Maybe rename the png file, replace Singin by Signin

Reply:
Has Changed.